examples/with-reason-react/src/server.js

```diff
@@ -13,7 +13,7 @@ server
   .use(express.static(process.env.RAZZLE_PUBLIC_DIR))
   .get('/*', (req, res) => {
     const markup = renderToString(
-      <App title="Welcome to Razzle Reason React" />
+      <App title="Welcome to Razzle Reason React" initialUrl={req.url.substring(1)} />
```

Comment:
This is kind of a hack to get rid of the initial "/", should I change it to something else?